.github/workflows/npm_publish.yml

```diff
@@ -0,0 +1,30 @@
+name: npm publish
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
```

Comment:
It's usually sufficient to use `ubuntu-latest` unless you specifically need this version of ubuntu.